bake/compose.go

```diff
@@ -185,7 +185,7 @@ func loadDotEnv(curenv map[string]string, workingDir string) (map[string]string,
 		return nil, err
 	}
 
-	envs, err := dotenv.UnmarshalBytes(dt)
+	envs, err := dotenv.UnmarshalBytesWithLookup(dt, nil)
```

Comment:
signature changed: https://github.com/compose-spec/compose-go/commit/784a7e77d00efbc046bc9b3c19b8a5936ffe1606#diff-4c44c44d24dee73102acafac9247f9522055c6f12509b32912544f239f314b8dL136-L139

Reply:
Will that be problematic (e.g. compose using both `buildx 0.10` (which doesn't have this patch) and `compose-go 1.9.0` ? Do we need (deprecated) compatibility wrappers in compose-go? ./cc @ndeloof

Reply:
I don't think so, compose-go in buildx is only used for bake and compose only use the `build` package in buildx afaik.